cmd/ike/session/session_client.go

```diff
@@ -1,7 +1,7 @@
 package session
 
 import (
-	"log"
+	logger "log"
```

Comment:
Different logger?

Reply:
One is on the client side and the other within the cluster. We can use the same though, won't harm

Reply:
Or is it...

Reply:
Fixed with #fd2c88f

Reply:
and bb11168